src/NHSD.BuyingCatalogue.Identity.Api/Services/ProfileService.cs

```diff
@@ -30,7 +30,8 @@ public sealed class ProfileService : IProfileService
                     }
                 },
                 {
-                    OrganisationFunction.Buyer,  new List<Claim>
+                    OrganisationFunction.Buyer, 
+                    new List<Claim>
```

Comment:
Sorry Simon I wasn't clear in regards to my spacing comment. I was referring to the line `new Claim(ApplicationClaimTypes.Ordering,ApplicationPermissions.Manage)`. With no space after the `.Ordering,` I find it difficult to read. Again this comes down to needing a review of our tooling e.g. Resharper settings

Reply:
Ok, Just fixing them now , not claims are in the list so adding the missing claims

Reply:
Ok will fix that , i  have a fix for the integration tests too.